superset-frontend/src/components/Form/FormItem.tsx

```diff
@@ -21,20 +21,23 @@ import Form, { FormItemProps } from 'antd/lib/form';
 import { styled } from '@superset-ui/core';
 
 const StyledItem = styled(Form.Item)`
-  .ant-form-item-label > label {
-    text-transform: uppercase;
-    font-size: ${({ theme }) => theme.typography.sizes.s}px;
-    color: ${({ theme }) => theme.colors.grayscale.base};
+  .ant-form-item-label {
+    padding-bottom: ${({ theme }) => theme.gridUnit}px;
```

Comment:
Just a nit. For brevity, as the theme is widely used you can wrap the style within 

```
${({ theme })} => `...`
```

Reply:
Done!